better_square_roots

```diff
@@ -0,0 +1,16 @@
+a = input("Please enter a positive number:")
+if not a.isdigit() or float(a) < 0:
+    print ("Sorry, that is not a positive number!")
+else:
+    x = float(N/2)
+    runs = 0
+    y = []
+
+    while x **2 > N + 0.001:
+        runs +=1
+        x = ((N / x)+x)/2
+        y.append(round(x,3))
+
+    print("The square root of " + str(N) + " is " + "%.2f" % x + ".")
+    print("We did " + str(runs) + " calculations to get this answer.")
+    print("These were our guesses: " + str(y))
```

Comment:
make sure this solution is the `square_roots.py` file and you're using descriptive variable names.